updating/migrating-to-multi-payload.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="migrating-clusters-to-multi-payload"]
+= Migrating a cluster to a heterogenous payload
+include::_attributes/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can migrate your current homogeneous cluster to a heterogeneous cluster by updating to a heterogeneous manifest-listed payload, allowing you to add mixed architecture workers for future use.
```

Comment:
We were suggested to use single-arch and multi-arch instead of homogeneous and heterogeneous respectively.

Reply:
> add mixed architecture workers for future use.

add multi-arch compute nodes workers for future use.


cc @kelbrown20 to have coherent wording across the doc regarding multi-arch compute nodes.